spec/features/expenses/views_all_expenses.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+feature "view all expenses" do
+  scenario "successfully" do
+    user = create(:user)
+    create(:budget, user: user)
+    expense1 = create(:expense)
```

Comment:
How is this expense associated with this user and budget?
Also, this spec doesn't make sense with regards to the previous spec: "user sees only their expenses".

Reply:
Mmm, it wasn't, and no it isn't needed, thanks for the pickup!
Done in sha: 5393979